Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -432,6 +434,42 @@ private void AddCargoContents(CargoShuttleComponent component, StationCargoOrder
         }
     }
 
+    /// <summary>
+    /// In this method we are printing and attaching order manifests to the orders.
+    /// </summary>
+    private void spawnAndAttachOrderManifest(EntityUid item, CargoOrderData order, EntityCoordinates coordinates, CargoShuttleComponent component) {
+
+        if (!_protoMan.TryIndex(order.ProductId, out CargoProductPrototype? prototype))
+            return;
+
+        // spawn a piece of paper.
+        var printed = EntityManager.SpawnEntity("Paper", coordinates);
+
+        if (!TryComp<PaperComponent>(printed, out var paper))
+            return;
+
+        // fill in the order data
+        var val = Loc.GetString("cargo-console-paper-print-name", ("orderNumber", order.OrderNumber));
+
+        MetaData(printed).EntityName = val;
+
+        _paperSystem.SetContent(printed, Loc.GetString(
+            "cargo-console-paper-print-text",
+            ("orderNumber", order.OrderNumber),
+            ("itemName", prototype.Name),
+            ("requester", order.Requester),
+            ("reason", order.Reason),
+            ("approverName", order.ApproverName),
+            ("approverJob", order.ApproverJob)),
+            paper);
+
+        // attempt to attach the label
+        if (TryComp<PaperLabelComponent>(item, out var label))
+        {
+            _slots.TryInsert(component.Owner, label.LabelSlot, printed, null);
+        }
```

Comment:
It's attaching the label to... the cargo shuttle?

Reply:
I'm still getting used to C# and the project itself. This piece of code is copied from CargoSystem.Telepad and I didn't understand why this is done like that as well. So I decided to leave it as it is (it worked).
Will fix in the next commit.